pallet-logion-loc/src/lib.rs

```diff
@@ -1285,5 +1326,19 @@ pub mod pallet {
                 || match &loc.requester { Account(requester_account) => *submitter == *requester_account, _ => false }
                 || Self::verified_issuers_by_loc(loc_id, submitter).is_some()
         }
+
+        fn apply_file_storage_fee(fee_payer: T::AccountId, sizes: Vec<u32>) -> DispatchResult {
+            let byte_fee: BalanceOf<T> = T::FileStorageByteFee::get().into();
+            let entry_fee: BalanceOf<T> = T::FileStorageEntryFee::get().into();
+            let tot_size: u32 = sizes.iter()
+                .fold(0, |tot, current| tot + current);
```

Comment:
Why not doing this one level up and pass the total size as a parameter to `apply_file_storage_fee`?

Reply:
Good point. Done.